l2tdevtools/review_helpers/review.py

```diff
@@ -183,7 +183,7 @@ def CreatePullRequest(self):
     """Creates a GitHub pull request.
 
     Returns:
-      bool: True if the create was successful.
+      bool: True if the pull request was created successfully.
```

Comment:
This isn't correct - this returns False if the pull request is created, but the review or assignment fails. Please fix the docstring, or refactor the method.